lib/routers/profile/person.js

```diff
@@ -49,6 +49,60 @@ const validateEmail = () => {
   };
 };
 
+const validatePassword = () => {
+  return (req, res, next) => {
+    const password = get(req, 'body.data.password');
+
+    if (!password) {
+      throw new BadRequestError('A password must be provided');
+    }
+
+    // todo: check password strength
+    if (password.length < 10) {
+      throw new BadRequestError('Password must be at least 10 characters');
+    }
+
+    if ((password.match(/[A-Z]/g) || []).length < 1) {
```

Comment:
```suggestion
    if (!password.match(/[A-Z]/g)) {
```

Reply:
Although it's arguably the case that we might want to preserve the counting in order to adjust up from 1.

Reply:
That's what I was thinking.

Reply:
Could stick some `const`s in for the counts?

Reply:
Good idea. That would at least serve as a clue to future me as to why the code is apparently needlessly verbose.